compiler-rt/lib/scudo/standalone/release.h

```diff
@@ -200,22 +208,16 @@ template <uptr StaticBufferCount, uptr StaticBufferSize> class BufferPool {
 class RegionPageMap {
 public:
   RegionPageMap()
-      : Regions(0),
-        NumCounters(0),
-        CounterSizeBitsLog(0),
-        CounterMask(0),
-        PackingRatioLog(0),
-        BitOffsetMask(0),
-        SizePerRegion(0),
-        BufferSize(0),
-        Buffer(nullptr) {}
+      : Regions(0), NumCounters(0), CounterSizeBitsLog(0), CounterMask(0),
+        PackingRatioLog(0), BitOffsetMask(0), SizePerRegion(0),
+        BufferNumElements(0), Buffer(nullptr) {}
```

Comment:
Just curious, are these lines formatted by clang-format? I think you didn't change the code here.

Reply:
I did change `BufferSize` to `BufferNumElements`, and the GH presubmit checks complained:
![Screenshot from 2023-09-27 15-55-16](https://github.com/llvm/llvm-project/assets/5803506/c70db366-c5b9-40a5-a519-8aaf038d99f2)
The fix was to run clang-format, yes